packages/bpk-component-chip-group/src/BpkChipGroup.tsx

```diff
@@ -70,131 +70,172 @@ export type ChipItem = {
   hidden?: boolean;
 } & SingleSelectChipItem;
 
+export type InternalProps = {
+  chipGroupClassNames: string
+};
+
 export type CommonProps = {
   label?: string;
   ariaLabel?: string;
-  type?: ChipGroupType;
   chipStyle?: ChipStyleType;
-  // only required when type = rail
-  leadingNudgerLabel?: string;
-  trailingNudgerLabel?: string;
+  chips: ChipItem[];
+  ariaMultiselectable?: boolean;
 };
 
-export type ChipGroupProps = {
-  chips: ChipItem[];
+export type RailChipGroupProps = {
   stickyChip?: ChipItem;
-  ariaMultiselectable?: boolean;
+  leadingNudgerLabel?: string;
+  trailingNudgerLabel?: string;
+} & CommonProps;
+
+export type WrapChipGroupProps = {
```

Comment:
There isn't anything specific in here but open question whether we like having a marker type to describe the scenario